js/reload_controller.js

```diff
@@ -37,13 +41,16 @@ ReloadController = function()
     'reloadUnpinnedOnly',
     'reloadAllRight',
     'reloadAllLeft',
+    'closeAllRight',
+    'closeAllLeft',
     'reloadStartup',
     'bypassCache',
     'version'
   ]
 
   chrome.storage.sync.get(settingsToFetch, settings => {
     this.cachedSettings.version = settings.version
+    this.cachedSettings.buttonDefaultAction = (typeof settings.buttonDefaultAction == 'undefined') ? "window" : settings.buttonDefaultAction
```

Comment:
```suggestion
    this.cachedSettings.buttonDefaultAction = (typeof settings.buttonDefaultAction == 'undefined') ? 'window' : settings.buttonDefaultAction
```
nit: single quote instead of double

Reply:
updated in my fork